### PR TITLE
Support graphql 0.11

### PR DESCRIPTION
**Pull request by danez**
This adds graphql 0.11 as valid version for the peer dependency.

### CODE REVIEW

**Comment by danez** [comment]
This works now.

File: resources/prepublish.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Because of a long-running npm issue (https://github.com/npm/npm/issues/3059)
 # prepublish runs after `npm install` and `npm pack`.
 # In order to only run prepublish before `npm publish`, we have to check argv.
-if node -e "process.exit(($npm_config_argv).original[0].indexOf('pu') === 0)"; then
+if node -e "process.exit(($npm_config_argv).original.length && ($npm_config_argv).original[0].indexOf('pu') === 0)"; then
```
Comment (danez) [inline resources/prepublish.sh]: This allows running simply `yarn` without argument

**Comment by danez** [comment]
Let me know if this is okay.

**Comment by gdi2290** [comment]
I also need this PR merged

**Comment by danez** [comment]
ping @wincent :)

**Comment by tlvenn** [comment]
@leebyron any chance to get this merged ?

Once this PR is fixed and a new release is cut, it will allow a new release of GraphiQL to depend on it so that none of its dependencies are pulling graphql on their own anymore.

Right now, the current version of this lib is depending on `"graphql-language-service-interface": "^0.1.13"` which lists `graphql@0.10.5` has its dependency which is problematic upstream as soon as you are depending on 0.11.x

> https://github.com/graphql/graphql-language-service/blob/de1d5a3bd755f43838432510bd2dde80a63a65fd/packages/interface/package.json#L29

Thanks a lot in advance.

**Comment by gpoitch** [comment]
Currently cannot run the latest version of graphiql with the latest version of graphql correctly without this update because it creates duplicate graphql dependencies in 0.10.x & 0.11.x

**Comment by pleunv** [comment]
You can temporarily work around this (if you use yarn) by adding graphql to your resolutions in `package.json`:

```json
"resolutions": {
  "graphql": "^0.11.7",
}
```

Which will force 0.11.7 to be installed for every dependency. I've noticed it's generally not a bad idea to always do this for graphql :).

**Comment by wibron** [comment]
Would be very nice to get this merged :+1:

**Comment by gdi2290** [comment]
how can we get this merged?

**Comment by tlvenn** [comment]
For reference @pleunv , I tried to use Yarn's selective dependency resolution to solve my issue with duplicated graphql.js and in my case, it did not work. Only working solution was to edit manually yarn.lock

@wincent @leebyron please save us ;)

**Comment by benjick** [comment]
Any update on this?

**Comment by pleunv** [comment]
@tlvenn That's really odd, are you on yarn 1.0+? Using forced resolution for a handful of deps and never had an issue with it yet.

**Comment by danez** [comment]
Also does not work for me on latest yarn. yarn always thinks the lock file is out of date, although it isn't.

**Comment by tlvenn** [comment]
@pleunv I updated Yarn to the 1.3 from 1.1 and this time it worked properly.

**Comment by IvanGoncharov** [comment]
@asiandrummer @wincent @leebyron Can you please review this PR?
It's forcing all projects that depend on GraphiQL either use `yarn` specific workaround or block dependencies updates.

**Comment by asiandrummer** [comment]
Whoa, sorry about going rogue on this! Merged - will do a release shortly.

**Comment by gdi2290** [comment]
🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉

**Comment by asiandrummer** [comment]
https://github.com/graphql/codemirror-graphql/releases/tag/v0.6.12 is out!

**Comment by pleunv** [comment]
Awesome, thanks!

**Comment by danez** [comment]
It seems 0.6.12 wasn't published on npmjs. :)

**Comment by IvanGoncharov** [comment]
@danez You are right 😞 
image

@asiandrummer Can you please publish NPM package?

**Comment by danez** [comment]
@IvanGoncharov The npmjs.org website is usually not a good indicator as it is cached for days. better to look at https://registry.npmjs.org/codemirror-graphql. Although not as readable, but the data is up2date there.

**Comment by IvanGoncharov** [comment]
@danez Great link 👍 

@asiandrummer @leebyron Can someone please publish `0.6.12` as NPM package?
Since merging this PR doesn't any solve problems until NPM package is released.

**Comment by IvanGoncharov** [comment]
@wincent You rock 👍 🎉 
`0.6.12` finally pushed to NPM so it's time to remove it from `resolutions` section.
image

**Pull request merged**